src/app/app.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { ProductComponent} from "./product/product.component";
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [RouterOutlet, ProductComponent, CommonModule],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.css'
+})
```

Comment:
```suggestion
@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrl: './app.component.css',
  standalone: true,
  imports: [RouterOutlet, ProductComponent, CommonModule],
})
```
Dans les bonnes pratiques de structure de fichier, on mets les standelone et import à la fin